package.xml

```diff
@@ -18,6 +18,8 @@
   <depend>gz-utils3</depend>
   <depend>libfreeimage-dev</depend>
   <depend>libgdal-dev</depend>
+  <depend>libgts</depend>
```

Comment:
I think `libgts` was removed in https://github.com/gazebosim/gz-common/commit/1f527012d21fd5afde75de8b49e0776e08f3cd6c

Reply:
Removed in [f1db533](https://github.com/gazebosim/gz-common/pull/615/commits/f1db533d430784d6fa63fbc0988a355f0ffa05a7).